pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -132,6 +132,28 @@ func (a *Bootstrap) Generate(dependencies asset.Parents) error {
 	if err != nil {
 		return err
 	}
+
+	// Check for optional platform specific files/units
+	platform := installConfig.Config.Platform.Name()
+	platformFilePath := fmt.Sprintf("bootstrap/%s/files", platform)
+	directory, err := data.Assets.Open(platformFilePath)
+	if directory != nil {
```

Comment:
Fix for checking err is in progress, will update and remove WIP when merged. https://github.com/openshift-metal3/kni-installer/pull/143